tests/rmutex/Makefile

```diff
@@ -1,5 +1,8 @@
 include ../Makefile.tests_common
 
+# MIPS based boards are black listed due to some issue with newlib and _Atomic
+BOARD_BLACKLIST := mips-malta pic32-clicker pic32-wifire
```

Comment:
note: this should be removed, if someone has solution for the MIPS toolchain. It seems that some include paths are mixed up and the mips-gcc cannot find some system header files?!

Reply:
The difference between master and this pr with `make info-build`:

``` diff
diff -u info-build-master  info-build-pr 
--- info-build-master   2018-06-19 13:47:58.983288813 +0200
+++ info-build-pr       2018-06-19 13:48:08.447320615 +0200
@@ -35,6 +35,8 @@
 FEATURES_CONFLICT_MSG: 
 
 INCLUDES: 
+       -isystem  
+       /opt/mips-mti-elf/2016.05-03/mips-mti-elf/include  
        -I/home/harter/work/git/RIOT/core/include  
        -I/home/harter/work/git/RIOT/drivers/include  
        -I/home/harter/work/git/RIOT/sys/include 
```

I also get warnings two times when doing info-build:

```
make -C tests/rmutex BOARD=pic32-wifire info-build > info-build-pr
/home/harter/work/git/RIOT/makefiles/libc/newlib.mk:51: newlib.h not found, guessing newlib include path from gcc path
/home/harter/work/git/RIOT/makefiles/libc/newlib.mk:51: newlib.h not found, guessing newlib include path from gcc path
```

Reply:
Where is your newlib installation located? (newlib.h)
What distro?

Reply:
Found that the two times  warning comes from `$(MAKE) info-boards-supported` being called during `make info-build`.

-----

I use ubuntu 16.04 and I installed the mips-mti-elf as explained in the Wiki I think:

```
$ /opt/mips-mti-elf/2016.05-03/bin/mips-mti-elf-gcc --version
mips-mti-elf-gcc (Codescape GNU Tools 2016.05-03 for MIPS MTI Bare Metal) 4.9.2
Copyright (C) 2014 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

And I have a only this `newlib.h` in mips-mti-elf directory, no `newlib-nano` directory:

`/opt/mips-mti-elf/2016.05-03/mips-mti-elf/include/newlib.h`